packages/bpk-component-image/src/withLazyLoading-test.js

```diff
@@ -20,30 +20,37 @@ import React from 'react';
 import { shallow, mount } from 'enzyme';
 import withLazyLoading from './withLazyLoading';
 
+const createDocumentMock = () => {
+  const mock = jest.fn();
+
+  mock.addEventListener = jest.fn();
+  mock.removeEventListener = jest.fn();
+  mock.documentElement = jest.fn();
+
+  return mock;
+};
+
 describe('withLazyLoading', () => {
   it('should return the original component', () => {
-    const documentMock = jest.fn();
+    const documentMock = createDocumentMock();
     const MockImageComponent = () => (<div>Fake Image</div>);
     const LazyLoadedImage = withLazyLoading(MockImageComponent, documentMock);
 
-    const lazyLoadedImage = shallow(<LazyLoadedImage />);
+    const lazyLoadedImage = shallow(<LazyLoadedImage />, { disableLifecycleMethods: true });
     expect(lazyLoadedImage.first().is(MockImageComponent));
   });
 
   it('should add inView prop', () => {
-    const documentMock = jest.fn();
+    const documentMock = createDocumentMock();
     const MockImageComponent = () => (<div>Fake Image</div>);
     const LazyLoadedImage = withLazyLoading(MockImageComponent, documentMock);
 
-    const lazyLoadedImage = shallow(<LazyLoadedImage />);
+    const lazyLoadedImage = shallow(<LazyLoadedImage />, { disableLifecycleMethods: true });
```

Comment:
Enzyme previously didn't call lifecycle methods such as `componentDidMount` and as such these tests relied on this behaviour. See [this](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#lifecycle-methods)